lib/config.py

```diff
@@ -11,8 +11,6 @@
 sentinel_config_file = os.environ.get('SENTINEL_CONFIG', default_sentinel_config)
 sentinel_cfg = DashConfig.tokenize(sentinel_config_file)
 sentinel_version = "1.3.0"
-min_dashd_proto_version_with_sentinel_ping = 70207
-
```

Comment:
```
/lib/config.py:15:1: E302 expected 2 blank lines, found 1
def get_dash_conf():
^
The command "find ./lib ./test ./bin -name \*.py -exec pycodestyle --show-source --ignore=E501,E402,E722,E129,W503,W504 {} +" exited with 1.
```
https://travis-ci.org/dashpay/sentinel/jobs/511543379#L518

Reply:
:+1: Fixed this